pkg/validator/models/rbac.go

```diff
@@ -11,3 +11,7 @@ type RBACRules struct {
 	Data interface{}
 	Type Rule
 }
+
+func GetServiceAccountIsRoleBound() Rule {
```

Comment:
Could we make it constant?

Reply:
If we want to make ServiceAccountIsRoleBound constant, we need to initialize it with some value, but I don't know which one

Reply:
Yes, it should be constant. Initialize it for example like this:
`var ServiceAccountIsRoleBound Rule = "ServiceAccountIsRoleBound"`